### ISSUE TITLE
[Feature request] Support Retina displays

### ISSUE THREAD

**Issue by i3games** [EmissionControl2/EmissionControl2#82]
On a Macbook with Retina display, the user interface is blurry / jagged.

<img width="757" alt="Screenshot 2020-11-14 at 21 20 27" src="https://user-images.githubusercontent.com/830492/99156237-8f512180-26bf-11eb-8436-8e16b52e94ca.png">

for comparison:
<img width="712" alt="Screenshot 2020-11-14 at 21 22 22" src="https://user-images.githubusercontent.com/830492/99156239-924c1200-26bf-11eb-840e-71078d5c8e26.png">

**Comment by rodneydup** [comment]
Hmm... I assume this happens when you use the font scale feature to enlarge the font size? It's probably just blowing up a smaller font to a large size instead of loading in the font at a larger resolution. I can probably improve this in the next release.

**Comment by i3games** [comment]
No, its the default view from initial startup on my Macbook Pro with Retina display. Font size scale is default 1.0.

**Comment by rodneydup** [comment]
Oh interesting. I will look into this. Thanks for pointing it out.
